ptbcontrib/log_forwarder/README.md

```diff
@@ -0,0 +1,14 @@
+# Logging handler for python-telegram-bot
+Extension to forward application logs to specific Telegram chats through a `logging.Handler` class.
+
+## Usage
+Initialize the log forwarder in your `on_startup` method:
+
+```python
+async def on_startup(app: Application):
```

Comment:
maybe also add the builder pattern below to emphasize that this should be passed to `post_init`?

Reply:
I have added the builder pattern.